src/Core/src/Platform/Android/LayoutViewGroup.cs

```diff
@@ -96,6 +97,43 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 			SetMeasuredDimension((int)platformWidth, (int)platformHeight);
 		}
 
+		Rectangle AdjustForSafeArea(Rectangle bounds)
```

Comment:
Can you try applying this logic to the `WrapperView` instead of the LayoutViewGroup